### ISSUE TITLE
I am trying to install ckanext-spatial on a ckan 2.10, python 3.9 on a devian 11 SO. The first problem I found was when I try to install the packages is that package libgeos-c1 in debian 11 is now libgeos-c1v5, also my so selects python-dev-is-python2 instead python-dev (This I don´t know if I may force choosing python3-dev). Once I've corrected this two issues it aparents to install fine the python packages and the ckanext-spatial plugin. The problems comes when I try to install the requirenments with pip install -r requirements.txt. Then it gives me the next error:

### ISSUE THREAD

**Issue by martabracho** [ckan/ckanext-spatial#315]

**Comment by martabracho** [comment]
I am trying to install ckanext-spatial on a ckan 2.10, python 3.9 on a devian 11 SO. The first problem I found was when I try to install the packages is that package libgeos-c1 in debian 11 is now libgeos-c1v5, also my so selects python-dev-is-python2 instead python-dev (This I don´t know if I may force choosing python3-dev). Once I've corrected this two issues it aparents to install fine the python packages and the ckanext-spatial plugin. The problems comes when I try to install the requirenments with pip install -r requirements.txt. Then it gives me the next error: proj executable not found. Please set the PROJ_DIR variable. I've tried to install pyproj in my /lib/ckan/default/lib/python3.9/site-packages and put the envioronment variables to:
export PROJ_DIR=/lib/ckan/default/lib/python3.9/site-packages/pyproj
export PROJ_LIBDIR=/lib/ckan/default/lib/python3.9/site-packages/pyproj.libs
export PROJ_INCDIR=/lib/ckan/default/lib/python3.9/site-packages/pyproj/proj_dir
but still gives me this error:  FileNotFoundError: [Errno 2] No such file or directory: '/lib/ckan/default/lib/python3.9/site-packages/pyproj/bin/proj'

**Comment by martabracho** [comment]
More information:
I've realized that PROJ and PYPROJ are defferent packages. I've installed both with:

sudo apt-get install proj-bin

and 

pip install pyproj

and then I had to run

pip install -r requirements.txt

with root user because it gave me an error of permission, and then it sais:

ERROR: Minimum supported PROJ version is 9.0.0, installed version is 7.2.1

**Comment by martabracho** [comment]
I've solved my problem. I installed PROJ version 9.0.0 form source and then compile it, because debian 11 does not install version 9.0.0 form the apt.

**Comment by markstuart** [comment]
Yeah, I have the same issue in the context of a Docker build. Using a python:3.10-slim-bullseye base image, if I `apt install proj-bin

**Comment by FedericOldani** [comment]
Hey, I'm just stuck on this problem. Can you write the code to build PROJ >=9.0.0 from Dockerfile?
Thank you

**Comment by jqnatividad** [comment]
Hi @amercader , we also encountered this on a CKAN 2.10 installation with a Ubuntu 22.04 (python 3.10) installation:

```
#45 263.8   Cloning https://github.com/pyproj4/pyproj.git (to revision main) to /tmp/pip-install-qnbv6t6q/pyproj_757501df313842ddbba93bd83eefd113
  #45 263.8   Running command git clone --filter=blob:none --quiet https://github.com/pyproj4/pyproj.git /tmp/pip-install-qnbv6t6q/pyproj_757501df313842ddbba93bd83eefd113
  #45 263.8   Resolved https://github.com/pyproj4/pyproj.git to commit 42aa92f489bd266749a44cb78101eae28e20f5c3
  #45 263.8   Installing build dependencies: started
  #45 263.8   Installing build dependencies: finished with status 'done'
  #45 263.8   Getting requirements to build wheel: started
  #45 263.8   Getting requirements to build wheel: finished with status 'error'
  #45 263.8   error: subprocess-exited-with-error
  #45 263.8
  #45 263.8   ×  Getting requirements to build wheel did not run successfully.
  #45 263.8   │  exit code: 1
  #45 263.8   ╰ ─ > [1 lines of output]
  #45 263.8       ERROR: Minimum supported PROJ version is 9.2.0, installed version is 8.2.1. For more information see: https://pyproj4.github.io/pyproj/stable/installation.html
  #45 263.8       [end of output]
  #45 263.8 (
```

Do we just need to build pyproj 9.2.0 from source?

If so, we'll pull together a PR to include it in the troubleshooting section of the docs.
https://docs.ckan.org/projects/ckanext-spatial/en/latest/install/#troubleshooting

cc @twdbben
